### ISSUE TITLE
[BUG] FCU and Autoflight 

### ISSUE THREAD

**Issue by A320pilotIRL** [flybywiresim/a32nx#358]
Real A320 pilot. FCU/Autoflight should be a dev priority. You shouldn’t need to push or pull the ALT knob to update the altitude the avionics recognize, the “blue” altitude should change instantly as the knob is rotated. I should never need to push or pull the ALT knob to change the altitude. As for Vertical Speed, you should pull the knob once to get into VS mode, then any time you rotate the knob (without pulling!) the VS should update. Any time the ALT knob is pulled, and a higher-than-current altitude is set, it should be THR CLB OP CLB. When capturing LOC or GS, it should go LOC* or GS* during the capture process. VNAV is of course unusable/nonexistent. Autothrust does not target the proper speeds when using managed speed during the approach phase (should always be slowing you to below next flap speed). Moving the thrust levers to  TOGA on an approach should activate go-around phase and guidance.  You should be able to go direct-to (DIR key on MCDU) all the proper fixes on the flight plan, currently only a few are selectable. You should also be able to enter speed and altitude restrictions by pressing the key to the right of the fix when you’ve typed the altitude/speed into the scratchpad. Shouldn’t go to lat rev. These are just a SMALL handful of the existing issues with the FCU and automation. None of this is your fault, MSFS half-baked the plane. 

The FCU and automation is MUCH more important than other issues like ECAM pages, voltages, sounds, screens, systems, etc. what matters most is that you can fly the plane like the real thing.

**Comment by Curtis-VL** [comment]
FMA is actively been worked on, I'll take a look through these issues you mentioned. 👍

**Comment by Curtis-VL** [comment]
To clarify to passers-by, these changes are relevant to the 'master' branch, not 'dev'!

**Comment by JackZ974** [comment]
Yeah, basically the whole Autopilot automation has basically to be recreated/ started from scratch, we all know that. 
For now the Airbus uses the standard (FSX Based) Autopilot, so there is lots of room for improvement, but don't expect too quick progresses/new features on that field, as it is something VERY complicated to implement properly. 
As a proof even hardcore addons on similar platforms (FSLABS, TOLISS, Flight Factor to name a few) are still struggling with some issues, aven after years of development...

So basically what can be done at first is marginally improve the logic of the FCU behaviour to make it resemble the real thing
But don't put expectations too high, at least at the beginning.

Another RL A320 Pilot.

**Comment by Curtis-VL** [comment]
Alright, finally gonna' tackle some of these bugs! There's a lot in this one issue so I'll try to categorize them!

- [ ] You shouldn’t need to push or pull the ALT knob to update the altitude the avionics recognize, the “blue” altitude should change instantly as the knob is rotated.

- [ ] As for Vertical Speed, you should pull the knob once to get into VS mode, then any time you rotate the knob (without pulling!) the VS should update.

- [ ] Any time the ALT knob is pulled, and a higher-than-current altitude is set, it should be THR CLB OP CLB.

- [ ] When capturing LOC or GS, it should go LOC* or GS* during the capture process.

- [ ] Autothrust does not target the proper speeds when using managed speed during the approach phase (should always be slowing you to below next flap speed).

- [ ] Moving the thrust levers to TOGA on an approach should activate go-around phase and guidance.

- [ ] You should be able to go direct-to (DIR key on MCDU) all the proper fixes on the flight plan, currently only a few are selectable.

- [ ] You should also be able to enter speed and altitude restrictions by pressing the key to the right of the fix when you’ve typed the altitude/speed into the scratchpad.

**Comment by Curtis-VL** [comment]
Might need to pull in the MCDU person to look through some of this too! 👍

**Comment by Benjozork** [comment]
@Curtis-VL could we have a tracking issue for this then ?

**Comment by Andreatat69** [comment]
Hi !! there is a reason when I turn on the external power FCU panal remain completely OFF ??

**Comment by MisterChocker** [comment]
issues mentioned in this issue have been fixed, closing
